test/test_jit.py

```diff
@@ -1876,6 +1876,57 @@ def test_print_graph_executor(meme):
         if not WINDOWS:
             self.assertExpected(captured[0])
 
+    def test_script_module(self):
+        class M1(torch.jit.ScriptModule):
+            def __init__(self):
+                super(M1, self).__init__(False)
+                self.weight = nn.Parameter(torch.randn(2))
+
+            @torch.jit.script_method
```

Comment:
What happens if I create a `ScriptModule` without any `script_method`s? It won't behave like a regular `nn.Module`, right?

Reply:
Not yet. We should discuss our strategy for gradually transitioning a module to a ScriptModule, and our mechanism for turning off the ScriptModule part for debugging.